components/html/modules/content-newsletter-text-only/_main-1.html

```diff
@@ -0,0 +1,49 @@
+<section class="intro">
+	<article>
+		<h1 class="intro__title">Title of Enewsletter Here</h1>
+			<p class="intro__issue">FALL 2017</p>
+			
+			<p class="intro__description">A brief introduction to the newsletter can go here if needed. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis vitae lacinia dui, ac lacinia dolor. Maecenas mattis nulla ac purus mattis, sit amet facilisis dolor luctus. Donec luctus convallis arcu.</p>
+	</article>
+</section>
+		
+<section class="section">
+	<article>
+		<h2 class="section__heading">Featured Stories</h2>
+		
+			<section class="story">
+				<h3 class="story__title">This title is for the featured story. Three lines used if necessary or needed.</h3>
+				<p class="story__subtitle">Subhead if needed</p>
+				<p class="story__teaser">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis vitae lacinia dui, ac lacinia dolor. Maecenas mattis nulla ac purus mattis, sit amet facilisis dolor luctus. Donec luctus convallis arcu. Nullam adipiscing, magna porttitor pellentesque ornare, magna tortor dignissim justo, vel fermentum massa neque sit amet urna.</p>
+			
+				<a href="#" class="button button--hover-light">Read More</a>
+			</section>
+	</article>
+</section>
+		
+<section class="section">
+	<article>
+		<h2 class="section__heading">Other Stories</h2>
+			
+       		<ul class="list">
+        		<li class="list__listItem">
+        		<span class="fa-stack fa-2x"><i class="fa fa-circle fa-stack-2x"></i><i class="fa fa-newspaper-o fa-stack-1x"></i></span>
+        		<span class="title-list">Lorem ipsum dolor sit amet, consectetur adipisicing elit.</span></li>
+        		
+        		<li class="list__listItem">
+        		<span class="fa-stack fa-2x"><i class="fa fa-circle fa-stack-2x"></i><i class="fa fa-newspaper-o fa-stack-1x"></i></span>
+        		<span class="title-list">Lorem ipsum dolor sit amet, consectetur adipisicing elit.</span></li>
+        		
+        		<li class="list__listItem">
+        		<span class="fa-stack fa-2x"><i class="fa fa-circle fa-stack-2x"></i><i class="fa fa-newspaper-o fa-stack-1x"></i></span>
+        		<span class="title-list">Lorem ipsum dolor sit amet, consectetur adipisicing elit.</span></li>
+    		</ul>
+	</article>
+</section>
```

Comment:
I think @bholaday and I talked about this before but I was thinking the structure of this section could look something like this:
```html
<h2 class="section__heading">Other Stories</h2>		
<ul class="section__list section__list-other">
    <li class="section__item section__item-other story story-other">
        <a href="#" class="story__link story__link-other">Lorem ipsum dolor sit amet, consectetur adipisicing elit.
    </li>
</ul>
```
The icons could then be added via CSS in the following manner:
```scss
.story{
    &__link{
        &-other{
            &::before{
                content:"\f1ea";
                font-family:$font-fontAwesome;
                display:block;
                float:left;
                /* other styles etc. */
            }
        }
    }
}
```
Thoughts @bholaday @jengarza ?

Reply:
@jacobvenable I've tried it without success. The icon has a circle behind it and trying to duplicate it with a border-radius hack only works as long as the element is square. It turns into an oval when the element gets taller (long article title). Hence the spans and flexbox. You could take a crack at it if you want a little challenge.

Reply:
How about this: https://jsfiddle.net/jLr6smdx/

Reply:
Nice work! Lets implement this, @jengarza

Reply:
Ok, hold on a sec. As I look at it more, the title's aren't centered vertically with the icons. centering them with flex gives you this: https://jsfiddle.net/f3ks5nan/

A long title brings the same oval problem as before. I'll mess with it now, but just wanted to get this comment up before we decide what to do.

Reply:
Comment storm! (sorry) Setting a `min-width:2em; min-height:2em;` fixed the oval. See: https://jsfiddle.net/f3ks5nan/1/

Reply:
You could also use flex to tell it not to grow or shrink: https://jsfiddle.net/jvenabl/f3ks5nan/2/

Reply:
Perfect! @jengarza Let's add this: https://jsfiddle.net/jvenabl/f3ks5nan/2/